### PR TITLE
164364254 Search Results Page

### PR DESCRIPTION
**Pull request by abayo-luc**
#### What does this PR do?
Add search results page
#### Description of Task to be completed?
• create a card for an article as well as user
• Search query input field
• Side component for relevant author and tas
#### How should this be manually tested?
* Fetch origin
* Checkout the branch `ft-search-results-page-164364254 `
* Open ```./pages/search-results.html#articles```
* and use the inspector to view the page with different devices.
#### What are the relevant pivotal tracker stories?
164364254
### Screenshots
screen shot 2019-03-08 at 12 56 22
screen shot 2019-03-08 at 12 56 33
screen shot 2019-03-08 at 12 57 08
screen shot 2019-03-08 at 12 57 21
screen shot 2019-03-08 at 12 57 29

### CODE REVIEW

**Review by YvesIraguha — APPROVED** [review]
@abayo-luc The page is responsive, and it meets specifications on the design.

File: css/main.css [hunk 1]
```diff
@@ -208,5 +405,119 @@ img.logo {
   img.logo {
     width: 5em;
     height: 5em; } }
+.search-results {
+  display: flex;
+  flex-direction: row;
```
Comment (YvesIraguha) [inline css/main.css:653]: There is a mixin defined for these two properties ` display: flex;
  flex-direction: row;`. I think for consistency you can use it `@include raw`

**Comment by espoirMur** [comment]
Sorry for this late review bro!!
It looks nice, 
What do you think about the space between author avatar and the article main picture here on mobile view?
Screen Shot 2019-03-12 at 10 07 07

File: pages/search-results.html [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+          </section>
+          <!-- display author -->
+          <section id="authors">
+            <div class="col-md-6 col-av-12 col-sm-12">
+              <div class="blog-card">
+                <div class="col-md-3">
+                  <img
+                    src="../img/user.jpg"
+                    alt=""
+                    style="width: 100px; height: 100px; border-radius: 50%"
```
Comment (espoirMur) [inline pages/search-results.html]: Can you please put the style in the sass file for consistency?

**Review by espoirMur — CHANGES_REQUESTED** [review]

**Review by espoirMur — APPROVED** [review]

**Review by puritybirir — APPROVED** [review]

**Pull request merged**